packages/EasyErrorHandler/src/Bridge/Symfony/Builder/ApiPlatformValidationErrorResponseBuilder.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Builder;
+
+use ApiPlatform\Core\Bridge\Symfony\Validator\Exception\ValidationException;
+use ApiPlatform\Core\Exception\InvalidArgumentException;
+use EonX\EasyErrorHandler\Builders\AbstractErrorResponseBuilder;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Serializer\Exception\MissingConstructorArgumentsException;
+use Symfony\Component\Serializer\Exception\UnexpectedValueException;
+use Symfony\Component\Validator\Constraints\NotNull;
+use Throwable;
+
+final class ApiPlatformValidationErrorResponseBuilder extends AbstractErrorResponseBuilder
+{
+    private const MESSAGE_PATTERN_INVALID_DATE = '/This value is not a valid date\/time\./';
+
+    private const MESSAGE_PATTERN_INVALID_IRI = '/Invalid IRI "(.+)"/';
+
+    private const MESSAGE_PATTERN_NESTED_DOCUMENTS_NOT_ALLOWED = '/Nested documents for attribute "(\w+)" ' .
+    'are not allowed/';
+
+    private const MESSAGE_PATTERN_NOT_IRI = '/Expected IRI or nested document for attribute "(\w+)", "(\w+)" given/';
+
+    private const MESSAGE_PATTERN_NO_PARAMETER = '/Cannot create an instance of [\"]?([\w\\\\]+)[\"]? from serialized' .
+    ' data because its constructor requires parameter "(\w+)" to be present/';
+
+    private const MESSAGE_PATTERN_TYPE_ERROR = '/The type of the "(\w+)" attribute must be "(\w+)", "(\w+)" given/';
+
+    public static function supports(Throwable $throwable): bool
+    {
+        $message = $throwable->getMessage();
+
+        return match ($throwable::class) {
+            InvalidArgumentException::class => \preg_match(self::MESSAGE_PATTERN_TYPE_ERROR, $message) === 1,
+            MissingConstructorArgumentsException::class =>
+                \preg_match(self::MESSAGE_PATTERN_NO_PARAMETER, $message) === 1,
+            UnexpectedValueException::class =>
+                \preg_match(self::MESSAGE_PATTERN_INVALID_DATE, $message) === 1 ||
+                \preg_match(self::MESSAGE_PATTERN_INVALID_IRI, $message) === 1 ||
+                \preg_match(self::MESSAGE_PATTERN_NESTED_DOCUMENTS_NOT_ALLOWED, $message) === 1 ||
+                \preg_match(self::MESSAGE_PATTERN_NOT_IRI, $message) === 1,
+            ValidationException::class => true,
```

Comment:
Do we really need to support `ValidationException` here? It seems we have all the required logic in `ApiPlatformValidationExceptionResponseBuilder`.

Reply:
No, removed